addons/sale/report/ir_actions_report_templates.xml

```diff
@@ -136,6 +136,7 @@
                         </t>
 
                         <t t-set="padding_class" t-value="line_padding and ('px-' + str(line_padding)) or ''"/>
+                        <t t-set="collapse_prices" t-value="current_subsection.collapse_prices or current_section.collapse_prices"/>
```

Comment:
probably will crash if `current_section` or `current_subsection` is `None` (my bad for the suggestion).
But value could probably be set in `if is_(sub)section` nodes instead no?

Reply:
Won't it still crash with your latest approach ? :thinking: